home-work3/utils.js

```diff
@@ -0,0 +1,113 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const path = require('path');
+const jsonexport = require('jsonexport');
+
+const filePath = path.join('.', 'events-data.csv');
+
+const getEvents = async () => {
+    return new Promise((resolve, reject) => {
+        const results = [];
+        fs.createReadStream(filePath)
+            .on('error', (err) => {
+                reject(err)
+            })
+            .pipe(csv())
+            .on('data', (data) => results.push(data))
```

Comment:
It would great if we could avoid writing all the data into memory and return data in chunks via streams.